modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -87,24 +90,34 @@ $ oc --context "${CTX_CLUSTER1}" apply -n istio-system -f https://raw.githubuser
 
 . Install {istio} on the West cluster:
 
-.. Create a YAML file named `istio-west.yaml` that defines an `Istio` resource on the West cluster.
+.. Save the IP address of the East-West gateway running in the East cluster by running the following command:
 +
-.Example resource file
-[source,yaml,subs="attributes,verbatim"]
+[source,terminal]
+----
+$ export DISCOVERY_ADDRESS=$(oc --context="${CTX_CLUSTER1}" \
+    -n istio-system get svc istio-eastwestgateway \
+    -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 ----
+
+.. Create a YAML file named `istio-west.yaml` that defines an `Istio` resource on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF > istio-west.yaml
 apiVersion: sailoperator.io/v1alpha1
 kind: Istio
 metadata:
   name: default
 spec:
   version: v${ISTIO_VERSION}
   namespace: istio-system
+  profile: remote
   values:
+    istiodRemote: 
+      injectionPath: /inject/cluster/cluster2/net/network1
     global:
-      meshID: mesh1
-      multiCluster:
-        clusterName: cluster2
-      network: network2
```

Comment:
This *is* required here.

Reply:
push containing this change was reverted.